sources/packages/backend/libs/sims-db/src/entities/application.model.ts

```diff
@@ -534,6 +534,12 @@ export interface ApplicationData {
    * Dependant details.
    */
   dependants?: unknown[];
+
+  /**
+   *
```

Comment:
Please remove the blank comment line.

Reply:
I was actually talking about 538.
![image](https://github.com/user-attachments/assets/bf6f1655-4aa1-4084-a568-d076129be0d2)